### ISSUE TITLE
False errors about line-endings

### ISSUE THREAD

**Issue by Alex-Sokolov** [Twiknight/eslint-plugin-vue#7]
I have .vue files and have eslint rule

```
"linebreak-style": ["error", "windows"],
```

In all files CRLF, but when run eslint it say that found LF in all files

P.S.: Using
- webpack 2.1.0-beta.25
- eslint: 3.8.1
- eslint-loader: 1.6.0
- eslint-plugin-vue: 0.1.1
